options/locale/locale_en-US.ini

```diff
@@ -737,6 +737,20 @@ issues.add_time_sum_to_small = No time was entered
 issues.cancel_tracking = Cancel
 issues.cancel_tracking_history = `cancelled time tracking %s`
 issues.time_spent_total = Total time spent
+issues.due_date = Due date
+issues.invalid_due_date_format = "Due date format is invalid, must be 'yyyy-mm-dd'."
+issues.error_modifying_due_date = "An error occured while modifying the due date."
+issues.error_removing_due_date = "An error occured while remvoing the due date."
+issues.due_date_form = "Due date, format yyyy-mm-dd"
```

Comment:
I think that
> format yyyy-mm-dd

could be confusing for users from non-english countries (like Germany) because modern browsers use for the UI the local date format (in Germany: dd.mm.yyyy) and as value `yyyy-mm-dd`. (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date#Value)

Reply:
Yes, but that value is used as a placeholder for the input. If the browser supports `input type date`, the browser does what you described thus not showing the placeholder at all (because there's already something inside that field).

Some screenshots with an empty input:

Firefox:
![bildschirmfoto von 2018-04-16 18-42-51](https://user-images.githubusercontent.com/13721712/38823228-1a1a1922-41a6-11e8-86bb-9daa5833dae8.png)

Chromium:
![bildschirmfoto von 2018-04-16 18-43-22](https://user-images.githubusercontent.com/13721712/38823234-201ffcf6-41a6-11e8-820d-4a0a91159400.png)

The format doesnt really seem to be an issue here.

Also, currently the date is not localized anywhere, so if we'd start doing that here we should do it everywhere which is totally out of scope for this pr.

Reply:
If the placeholder isn't shown at all it could be removed to reduce translation effort.